### ISSUE TITLE
StatusBar on Android Platform doesn't adapt properly

### ISSUE THREAD

**Issue by rllyhz** [rllyhz/aplikasi-KEM#3]
The statusbar background color and the content does not show properly due to lack of color contrast to the `Window` background color or the `AppBar` color.
